ui/admin/setup-edit.php

```diff
@@ -134,6 +134,9 @@
     if ( empty( $field_type_fields ) )
         $no_additional[] = $field_type;
 }
+
+// Make use of the WP core meta box functionality
+do_action( 'add_meta_boxes', 'pods_edit', $pod );
```

Comment:
Needs inline hook docs https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters